contracts/src/base/registry/facets/distribution/v2/RewardsDistributionStorage.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.18;
+
+// interfaces
+
+// libraries
+import {StakingRewards} from "./StakingRewards.sol";
+
+// contracts
+
+library RewardsDistributionStorage {
+  // keccak256(abi.encode(uint256(keccak256("facets.registry.rewards.distribution.v2.storage")) - 1)) & ~bytes32(uint256(0xff))
+  bytes32 internal constant STORAGE_SLOT =
+    0x9aed53e346ef79853612b4c863c4eb308de8a5e84e0fd7d00dad88eb5ff1ea00;
+
```

Comment:
I would define layout here since it's what we will be storing

Reply:
`StakingRewards` is self-contained with its minimal storage layout. `StakingRewards.Layout` is nested in `RewardsDistributionStorage.Layout`.